pom.xml

```diff
@@ -412,6 +412,14 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>3.2.0</version>
+                    <configuration>
+                        <addDefaultExcludes>false</addDefaultExcludes>
```

Comment:
I assume this is because of the various `git` paths etc.

Reply:
https://maven.apache.org/plugins/maven-resources-plugin/resources-mojo.html#addDefaultExcludes

I've added it mostly because of `BitKeeper ` as the other directories were already renamed somehow. I was just surprised that they were ignored in the first place. If needed, I can just create more mappings and remove this setting.

Reply:
I am confused. So even with the `addDefaultExcludes=false` the plugin actually moves/renames some of these entries but not all ?

Reply:
>I am confused. So even with the addDefaultExcludes=false the plugin actually moves/renames some of these entries but not all ?

No, without `addDefaultExcludes=false` the files that are matching these patterns are completely ignored and are not being copied to `target` directory:
 ```
Misc: **/*~, **/#*#, **/.#*, **/%*%, **/._*
CVS: **/CVS, **/CVS/**, **/.cvsignore
RCS: **/RCS, **/RCS/**
SCCS: **/SCCS, **/SCCS/**
VSSercer: **/vssver.scc
MKS: **/project.pj
SVN: **/.svn, **/.svn/**
GNU: **/.arch-ids, **/.arch-ids/**
Bazaar: **/.bzr, **/.bzr/**
SurroundSCM: **/.MySCMServerInfo
Mac: **/.DS_Store
Serena Dimension: **/.metadata, **/.metadata/**
Mercurial: **/.hg, **/.hg/**
GIT: **/.git, **/.gitignore, **/.gitattributes, **/.git/**
Bitkeeper: **/BitKeeper, **/BitKeeper/**, **/ChangeSet, **/ChangeSet/**
Darcs: **/_darcs, **/_darcs/**, **/.darcsrepo, **/.darcsrepo/****/-darcs-backup*, **/.darcs-temp-mail
 ```

Reply:
Okay, I understand now. With `addDefaultExcludes=false` the plugin will copy everything from the resources directory to the output directory.